index.html

```diff
@@ -23,29 +23,227 @@
                 </a>
             </div>
         </nav>
-    </header>
-    <main class="main">
-        <section class="headline">
-            <h2 class="headline-text">I’m Belay <br> Glad to see <br> you! </h2>
-            <p class="description-text"> I can help you build a product , feature or website
+        <div class="headline">
+          <div class="headline-container">
+            <div class="card-header">
+              <h2 class="headline-text">I’m Belay <br> Glad to see <br> you! </h2>
+              <p class="description-text"> I can help you build a product , feature or website
                 Look through some of my work and experience! If you
                 like what you see and have a project you need coded,
                 don’t hestiate to contact me.
-            </p>
-            <div>
-                <p class="social-icons-header">Let's Connect</p>
-                <ul class="social-icons">
+              </p>
+            </div>
+              <div class="card-body">
+                <div class="social-liks">
+                  <p class="social-icons-header">Let's Connect</p>

```

Comment:
Please note that this should be a link tag `<a>` which lets you connect o other people. It ideally should open your contact details or social media platforms.